### PR TITLE
Updates to Privacy Sandbox posts

### PR DESCRIPTION
**Pull request by samdutton**
Updates to origin trial status, naming and minor tweaks to wording and layout.

Requires approval before merging.

@rowan-m @maudnals fyi

### CODE REVIEW

File: src/site/content/en/blog/floc/index.md
Comment (mihajlija) [inline src/site/content/en/blog/floc/index.md:28]:
```suggestion
During the initial FLoC trial, a page visit was only included in the browser's FLoC computation 
```

**Review by mihajlija — APPROVED** [review]

File: src/site/content/en/blog/digging-into-the-privacy-sandbox/index.md
Comment (maudnals) [inline src/site/content/en/blog/digging-into-the-privacy-sandbox/index.md:99]: Suggestion to make this easier to maintain:

```suggestion
This API supports **click-through** and **view-through** measurement.

Other features in this API include cross-device attribution reporting and app-to-web attribution reporting. 
```

File: src/site/content/en/blog/digging-into-the-privacy-sandbox/index.md
Comment (maudnals) [inline src/site/content/en/blog/digging-into-the-privacy-sandbox/index.md:107]:
```suggestion
Both report types can be used simultaneously: they're complementary. 

Learn more about the status of these features and how to try this API [here](https://developer.chrome.com/docs/privacy-sandbox/attribution-reporting-introduction/#status).
```



---

**Review by maudnals — COMMENTED** [review]
LGTM %comments

**Comment by mihajlija** [comment]
@maudnals  CLA is bugging out about your commits

**Review by samthor — APPROVED** [review]

**Review by jpmedley — APPROVED** [review]

**Comment by samdutton** [comment]
Just to iterate, this requires internal sign-off and approval before merging (which is in progress).

**Comment by samdutton** [comment]
> @maudnals CLA is bugging out about your commits

@maudnals Hi Maud — not sure what's going on, but could you check and reply with `@googlebot I fixed it`. Thanks!

**Comment by samdutton** [comment]
@googlebot I fixed it.

File: src/site/content/en/blog/digging-into-the-privacy-sandbox/index.md
Comment (maudnals) [inline src/site/content/en/blog/digging-into-the-privacy-sandbox/index.md:7]:
```suggestion
updated: 2021-09-03
```

File: src/site/content/en/blog/digging-into-the-privacy-sandbox/index.md
Comment (maudnals) [inline src/site/content/en/blog/digging-into-the-privacy-sandbox/index.md:78]: Minor issue because that links redirects to this one anyway, but:
```suggestion
* [Attribution Reporting](https://github.com/WICG/conversion-measurement-api)<br>
```

**Review by maudnals — APPROVED** [review]
LGTM! (minor comments)

**Comment by samdutton** [comment]
Content approved.

**Pull request merged**
